tests/integration/test_fim/test_files/test_max_files_per_second/test_max_files_per_second.py

```diff
@@ -68,7 +68,7 @@ def test_max_files_per_second(inode_collision, get_configuration, configure_envi
     for i in range(n_files_to_create):
         fim.create_file(fim.REGULAR, test_directories[0], f'test_{i}', content='')
 
-    extra_timeout = n_files_to_create / max_files_per_second
+    extra_timeout = (n_files_to_create / max_files_per_second) + global_parameters.default_timeout
```

Comment:
The extra timeout is only required in case `inode_collision` is `True`, could we make this explicit by doing something like this?
```suggestion
    extra_timeout = n_files_to_create / max_files_per_second
    if inode_collision:
        extra_timeout += global_parameters.default_timeout
```

Reply:
Done in 53db7283792a368b061a52f8b2065dd30b3ce9b6